setup.py

```diff
@@ -45,7 +45,7 @@
         # grpc.Channel.close() method isn't added until 1.32.0.
         # https://github.com/grpc/grpc/pull/15254
         "grpcio >= 1.32.0, < 2.0dev",
-        "pyarrow >= 1.0.0, < 3.0dev",
+        "pyarrow >= 1.0.0, < 4.0dev",
```

Comment:
We actually need `pyarrow==3.0.0` to support Python 3.9, meaning that the lower bound should be bumped, too.